src/libecalc/domain/infrastructure/energy_components/asset/asset.py

```diff
@@ -12,15 +10,20 @@
 
 
 class Asset(Component, EnergyComponent):
+    def __init__(
+        self,
+        name: ComponentNameStr,
```

Comment:
this is pydantic validation, can use string here I guess?

Reply:
You are right, updated now